src/coreclr/src/vm/dwbucketmanager.hpp

```diff
@@ -475,8 +475,8 @@ void BaseBucketParamsManager::GetAppVersion(__out_ecount(maxLength) WCHAR* targe
     CONTRACTL
     {
         NOTHROW;
-        GC_NOTRIGGER;
-        MODE_ANY;
+        GC_TRIGGERS;
```

Comment:
Is `GC_TRIGGERS` appropriate here although `GCX_PREEMP()` is no longer invoked here?

Reply:
There doesn't seem to be anything in this method that could trigger a GC.